ocaml/xapi/test_vm_check_operation_error.ml

```diff
@@ -0,0 +1,43 @@
+
+let with_test_vm f =
+  let __context = Mock.make_context_with_new_db "Mock context" in
+  let vm_ref = Test_common.make_vm ~__context () in
+  f __context vm_ref
+
+(* CA-255128 Xapi_vm_lifecycle.check_operation_error should not throw an
+   exception when one of the nullable fields in the VM record are null. *)
```

Comment:
Ah, I see. In that case, I suggest
"...should not throw an exception even when all the VM record's nullable fields are null and the VM has a VBD with a null VDI."